.travis.yml

```diff
@@ -15,16 +15,15 @@ before_install:
-  - secure: pARLTGmAkJDH7UnXFjuC4pEuu5hAh7wlq27/X0ftJb6pFHM7rlXC8AruLXN9zahRy0/rCnaM6he+pAkRvWRSmU60FBV1t61yX0xRaAUumoDCSUWxEpW40upv/9cNJSPnwbF1YpCZFI5dq2ybX+ORy4VKBjpiKkR/aOgJSXzhF9VYhRUcrr6hIxpYPDSSMWs3PyYuSpT/+gNA4rkRyhVULVw/nsbolnTA65OExdcjbV629PPsTdJ9LGZM0Mite7v3RIo1eJdST8TP4tdIkxUF3BAH/LHqVdDoHJIVr6sXj704KM5PA3t4cnD615A0i36iAh3One4ntMvdbUTPRxy1WVC1W06CsZ2nb4FORGIwaE3i1d9BFaglOVgsc3yq44fgwHjqSntvEmCK/Vhve9ynRnHM2bla5mtfN/XlZZHu2N1Bzciwys5+QKEcFn6YLs9KnZFG5elIPgHWFx0JL4Zxo7rWhz9vv3327w/lJlE+nQ/DPup1Gmr5k9K6/FN71Aelntt2eWxfib9oYa+GaSZDOrZ7uzO+RXMehrZsKWNj9pogSPYvYf1Igpbt7XYmReVCau5UMI2to5w3EkUB26IydALDqSJZvLhxpw55M5x5IKZGgM9++WQ6Qjnxa7Do02osaWT076/9dYHBxPPlX0qEupJrpXfeYd81YTN5JwR8h2g=
-  - secure: ynS2Ru1UeHat3fzKx09dRiP8fjdtlQQpaeGEvz6MJGdJcoEwSZEFCjQzOWGWC5ZDY4CQUMjpUXe5s8xqHCXkW0UOAmyCNJbQPjpWExvyaYW3M+aEolFx0UikZNnrSTj4mBY9N5qKcIwn1sni6VUv11nLOH733n7n4lzV+tqJYITrhZ9NzkAO23goJFl4FNQi7uTM3T0ymEl1GSoWqb9HZMf8CCWXxig9XYDcN/37qC70QfBrwkXYNAWASwvJ4Rwkr5Qwo3TP16f6lkb1v+tZ+wu3GdH7C+mn+pTj59m9oolNzJ0DIWumN+TOy3wO+LKQqk0zj9Wpceljb0Y+tqmjQfyHobDlha5N3RarhyHFK3ROBr+WRpxYz7E575PGk9/Xg/r3x8qJNexjfjErJv7VmTbgHPG874cJw8S4t2ZLoKJa7/lZjOp3eLlbi/LTn+rqHLnMEepjHV0TfCtsiiUgzoTOwTOSQMeAKERpdmhl8d/Xcd+bP7UO5bJ1oo/bfDJOs/AXoYO6qN3qfq7nYLtZljL5xf60w8cXMKPSkoYUUabEMt/WrQ97wcPyaaodmOdUoa6RZmTA35oT/H6cIxqrreWilN5f/q85rRq+RLf2tCcqISxqWSmD08S/lTl59PSvodiWBylAkiI2/NZxrF/Tej2jBYZUssnPj5jUCAQheQU=
+  - SPHERE_PROJECT_KEY=sphere-product-type-import-test
+  - secure: pQ81rNbn9gaF71h9NyYCVUTF/FeWxRFA1BP1aF2XjMu8bZ8yMCR1Jdi98qM8MY90NIMqF1ECc65cPIleBfsrpg7JQKKd+tSxVRkkZVoam3XQnvM6BCF3ZrN3CDuq7c2WPTEivKzHZofCrDgmumaqIqYynaCqYUjtchPHkO2QSY0OZ6W8ee8bvFkQvkIZL3BkkuUwum4PuZNV60saLyg+OyD2XKXBp8u78RNbNu5fmb4eIk3ogPNvKnN0TqgjRa3CkIrctcduadGvLwh3aHNUWyScUjEd2pe9R7i9jFpVQtka5eABUebR/daMBr5M/S06EIQiYHbC4Gq2ojBVNsNpbUHL+AuAnduLn1FSFZigN5sigLwDBgRD0tNVx32Aju2yyhwGwyQq/I2AjKw/VGC63Rvo7IGNhdBriZalZ4kv9V8vz5C1dbePP1u+Sm9IFb8yA6O7HzZ/aV6xh3Yf/MCZm4RFqhVQbkib0rR95ib1U/GQ2R7N5Es9o2E8sX8WFszR6n0k4NF7JsQS7roQRDDVfkQxedyfnb6xD3TfYHk25dypiqSFZ8lKByLstzihU9WbNah+MSfj73uOK+olTEwAh3FiV9bhvgqV5FGGgCZ0QLgEePBZlvtImqJ7wR5/22cPMFLo/LKHcZPbEcx2v/GzaIApEXQLsORMkFGS9RS5A9A=
+  - secure: XO7/BiKlqhy8p7/9g4w9JAypp3t4NGDWxcbIu0jhl+UjqUKiMY6Qjq4h3w+cZC2SeX+FnxIkzdPYFnscAUmsMnRUXdGQMqegwNiCoeybQjIBD5J9+DHMiEOZoh5liJYkApk/O+sYG9XbPa89PoyoTxTS2MLbQOVmqrpJtpDWb6hOjl39x0iUDqO6ZkPTzWGiQcCij5BmAQOoHalgSsbdshaYJgoOiCVmKmj72aPplgvq3Ob1vgP/NTh8JesXeCNiN9cdP6kCb6SYQtioED9oH8D6t4h9qJUxIvf30+GA1RXXFkyMwym37SU10jotyhWmnjKE+7Q6xaQGpu7XfyIzcl98+X0bbl/LoAmBBtQBFHAhF5UyBDrpmzzRs+zi5NGHPHpuGUHbPawmpjFw/QVQIjWxw3zBkbjGfIQsPqlHMmaGRTrQ4d72JQXk/Kk1dlYLchCVl9gfA0BwO0EVGl3qdxf+TSAgLoOqbqhdSc8KfES7ZN8keViwxtV6fqDkwkubh9eX0H/iIfIL3tUH8Au9dFBrxGpNkabridqWwV070UWDKXHfrn6qhnYfumRZfDrOVvMhopUze0O/saxC9R9daEhsOZaNBVXNR8+OkGJu3KWClSZaowX+UdANItU8YXZYCa+4rUA74Q1H3X9JqIjfJnNg2aJTJI8gCAJgN88Vxsg=
```

Comment:
Same thing here regarding the encrypted variables

![image](https://cloud.githubusercontent.com/assets/1110551/19379561/e4fc115a-91f2-11e6-974a-8b6b63ff50f4.png)